### PR TITLE
feat(web-iconics) : implement Team Card component 

### PR DESCRIPTION
**Pull request by zerihunMoges**
Screenshot 2023-04-25 053654

implemented Team card component

### CODE REVIEW

File: Web/starter_project_iconics/pages/index.tsx [hunk 1]
```diff
@@ -1,124 +1,23 @@
...
-          </h2>
-          <p
-            className={`${inter.className} m-0 max-w-[30ch] text-sm opacity-50`}
-          >
-            Instantly deploy your Next.js site to a shareable URL with Vercel.
-          </p>
-        </a>
-      </div>
-    </main>
+    <div className="flex p-8">
```
Comment (sam5235) [inline Web/starter_project_iconics/pages/index.tsx:16]: If the repetition is needed you can use array.map() function in js...change it.

File: Web/starter_project_iconics/components/team/TeamCard.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    name: string
+    job: string
+    description: string
+    instagram?: string
+     linkedin?: string
+     facebook?: string
+     avatar?: string
+}
+
+function TeamCard({name, job, description, avatar = 'https://www.dmarge.com/wp-content/uploads/2021/01/dwayne-the-rock-.jpg', instagram, linkedin, facebook}: TeamCardProps) {
```
Comment (sam5235) [inline Web/starter_project_iconics/components/team/TeamCard.tsx:12]: Why you need the avatar value here, not good practice, also you doubled it.

File: Web/starter_project_iconics/pages/index.tsx (hunk 1 above)
Comment (zerihunMoges) [reply Web/starter_project_iconics/pages/index.tsx]: > If the repetition is needed you can use array.map() function in js...change it.

sorry that code was for test , I have removed it now

File: Web/starter_project_iconics/components/team/TeamCard.tsx (hunk 2 above)
Comment (zerihunMoges) [reply Web/starter_project_iconics/components/team/TeamCard.tsx:12]: ok

Comment (zerihunMoges) [reply Web/starter_project_iconics/components/team/TeamCard.tsx]: I have removed it, I wanted to set it as the default value

File: Web/starter_project_iconics/pages/index.tsx [hunk 3]
```diff
@@ -1,124 +1,23 @@
-import Image from 'next/image'
+import TeamCard from '@/components/team/TeamCard'
 import { Inter } from 'next/font/google'
```
[the remark this answers is not in the document]
Comment (zerihunMoges) [reply Web/starter_project_iconics/pages/index.tsx]: this was also for testing , i have removed it now , I will rebase and push the new code now

**Review by sam5235 — APPROVED** [review]

**Review by sam5235 — APPROVED** [review]

**Review by sam5235 — APPROVED** [review]

File: Web/starter_project_iconics/public/landing image area.png
Comment (Adona12) [inline Web/starter_project_iconics/public/landing image area.png:1]: Let's stick to kebab case

**Review by Adona12 — APPROVED** [review]
LGTM

**Pull request merged**